app/models/ems_refresh.rb

```diff
@@ -20,6 +20,10 @@ def self.debug_trace
     Settings.ems_refresh[:debug_trace]
   end
 
+  def self.max_targets_uniq_threshold
+    Settings.ems_refresh[:max_targets_uniq_threshold] || 1_000
```

Comment:
Prefer `Settings.ems_refresh.max_targets_uniq_threshold` and get rid of the `|| 1000`, because we will always have a value.

Reply:
On that note, why make it configurable at all?

Reply:
hm, right, maybe this config doesn't make that much of a sense